torch/_dynamo/variables/higher_order_ops.py

```diff
@@ -2452,14 +2454,11 @@ def maybe_positional_arg_names(func):
     result = []
     if not hasattr(func, "get_function"):
         return None
-    try:
-        fn = func.get_function()
-    except (Unsupported, NotImplementedError):
-        return None
     try:
         sig = inspect.signature(func.get_function())
-    except ValueError:
+    except (Unsupported, NotImplementedError, ValueError):
```

Comment:
technically, the behavior is slightly different as `NotImplementedError` on inspect.signature will get caught where it wasn't before.